### PR TITLE
Reference Documentation for Kubernetes Components and Tools for 1.17

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 1]
```diff
@@ -913,7 +913,7 @@ kube-apiserver [flags]
       <td colspan="2">--shutdown-delay-duration duration</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Time to delay the termination. During that time the server keeps serving requests normally and /healthz returns success, but /ready immediately returns failure. Graceful termination starts after this delay has elapsed. This can be used to allow load balancer to stop sending traffic to this server.</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">Time to delay the termination. During that time the server keeps serving requests normally and /healthz returns success, but /readzy immediately returns failure. Graceful termination starts after this delay has elapsed. This can be used to allow load balancer to stop sending traffic to this server.</td>
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kube-apiserver.md]: `readzy` looks like a typo; does that need fixing upstream?

File: content/en/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -69,14 +62,7 @@ kubectl [flags]
...
     <tr>
-      <td colspan="2">--boot-id-file string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/proc/sys/kernel/random/boot_id"</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of files to check for boot-id. Use the first one that exists.</td>
-    </tr>
-
-    <tr>
-      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "~/.kube/http-cache"</td>
+      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/Users/dsatya/.kube/http-cache"</td>
```
Comment (neolit123) [inline content/en/docs/reference/kubectl/kubectl.md]: better to have something like `/home/username/.kube...`

**Comment by neolit123** [comment]
i think for kubeadm we had more changes to the reference documentation, but i can't remember what else.. hm

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-18010--kubernetes-io-vnext-staging.netlify.com/docs/reference/kubectl/kubectl/

https://deploy-preview-18010--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/kube-apiserver/

https://deploy-preview-18010--kubernetes-io-vnext-staging.netlify.com/docs/reference/setup-tools/kubeadm/kubeadm-join/

/* This needs to be manually generated(diff'ed from cmd), if there are changes for the 1.17 release */
https://deploy-preview-18010--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/kubelet/

File: content/en/docs/reference/kubectl/kubectl.md (hunk 2 above)
Comment (kbhawkey) [reply content/en/docs/reference/kubectl/kubectl.md]: Any idea how this username was inserted into the output? Perhaps check the proposed output against the current (v1.16) page.

Comment (tengqm) [reply content/en/docs/reference/kubectl/kubectl.md]: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go#L57
and
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go#L326

Comment (daminisatya) [reply content/en/docs/reference/kubectl/kubectl.md]: @neolit123 Can I apply this manually?

**Comment by daminisatya** [comment]
@kbhawkey Can you help on how I can manually generate this page? 

https://deploy-preview-18010--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/kubelet/

File: content/en/docs/reference/kubectl/kubectl.md (hunk 2 above)
Comment (neolit123) [reply content/en/docs/reference/kubectl/kubectl.md]: @daminisatya yes

File: content/en/docs/reference/command-line-tools-reference/kube-apiserver.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/kube-apiserver.md]: Turns out that, actually, @sftim already did.
@daminisatya did you have revision https://github.com/kubernetes/kubernetes/commit/70132b0f130acc0bed193d9ba59dd186f0e634cf checked out when you generated the reference documentation?

File: content/en/docs/reference/kubectl/kubectl.md [hunk 3]
```diff
@@ -69,14 +62,7 @@ kubectl [flags]
...
     <tr>
-      <td colspan="2">--boot-id-file string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/proc/sys/kernel/random/boot_id"</td>
-    </tr>
-    <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of files to check for boot-id. Use the first one that exists.</td>
-    </tr>
-
-    <tr>
-      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "~/.kube/http-cache"</td>
+      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/Users/username/.kube/http-cache"</td>
```
Comment (sftim) [inline content/en/docs/reference/kubectl/kubectl.md]:
```suggestion
      <td colspan="2">--cache-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "~/.kube/http-cache"</td>
```

File: content/en/docs/reference/command-line-tools-reference/kube-apiserver.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/reference/command-line-tools-reference/kube-apiserver.md]: If I clone `kubernetes/kubernetes` and check out release-1.17 then run:
```bash
git annotate ./staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go | grep "returns success, but /r.*"
```
I see the fixed wording with `/readyz`.

Puzzled.

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/kube-apiserver.md]: And ... it was fixed by you, @sftim :)

https://github.com/kubernetes/kubernetes/commit/bbccdcd8531a6377c1fe2937aef24f61caef43c4

**Pull request merged**